com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/XRMirrorView.hlsl

```diff
@@ -1,11 +1,11 @@
 #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl"
 
-TEXTURE2D_X(_BlitTexture);
+TEXTURE2D_X(_SourceTex);
```

Comment:
i wonder if it has an impact for our users that we are changing the name of those variables. what do you think?

Reply:
This was really used only internally so there should be 0 impacts for users.